core/app/models/spree/tax/item_adjuster.rb

```diff
@@ -13,6 +13,7 @@ def initialize(item, options = {})
         @order = @item.order
         # set instance variable so `TaxRate.match` is only called when necessary
         @rates_for_order_zone = options[:rates_for_order_zone]
+        @tax_adjustments_destroyed = options[:tax_adjustments_destroyed]
```

Comment:
The name `tax_adjustments_destroyed` is a little confusing to me.

How about something like `:skip_destroy_adjustments`?